ask/src/main/java/com/vistrav/ask/Ask.java

```diff
@@ -119,6 +119,8 @@ public Ask when(@Nullable Permission permission) {
         void granted(List<String> permissions);
 
         void denied(List<String> permissions);
+
+        void onAllPermissionsGranted(boolean allGranted);
```

Comment:
this looks good but can you change the method name to `grantedAll` instead of `onAllPermissionsGranted` ?